spec/mailers/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe UserMailer, type: :mailer do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think we can actually remove this pending example and keep the blank test file for ancestors 😄